src/rollupToVinyl.ts

```diff
@@ -10,35 +10,60 @@ interface VinylPluginError {
   columnNumber?: number;
 }
 
+function isEmittedAsset(
+  chunkOrAsset: rollup.OutputChunk | rollup.OutputAsset,
+): chunkOrAsset is rollup.OutputAsset {
+  return (chunkOrAsset as rollup.OutputAsset).isAsset;
```

Comment:
Coerce the `undefined` to boolean.
```suggestion
  return !!(chunkOrAsset as rollup.OutputAsset).isAsset;
```